pangeo_forge/recipe.py

```diff
@@ -321,18 +372,75 @@ def region_for_chunk(self, chunk_key):
         # return a dict suitable to pass to xr.to_zarr(region=...)
         # specifies where in the overall array to put this chunk's data
         stride = self.nitems_per_input * self.inputs_per_chunk
-        start = chunk_key * stride
+        start = self.chunk_position(chunk_key) * stride
         region_slice = slice(start, start + self.nitems_for_chunk(chunk_key))
         return {self.sequence_dim: region_slice}
 
-    def sequence_len(self):
-        # tells the total size of dataset along the sequence dimension
-        return sum([self.nitems_for_chunk(k) for k in self.iter_chunks()])
-
     def sequence_chunks(self):
         # chunking
         return {self.sequence_dim: self.inputs_per_chunk * self.nitems_per_input}
 
     def iter_chunks(self):
         for k in self._chunks_inputs:
             yield k
+
+
+@dataclass
+class NetCDFtoZarrSequentialRecipe(NetCDFtoZarrRecipe):
```

Comment:
Thoughts on how to document this? I guess sphinx will pick up the parent doctoring. But the most important thing here is `input_urls`, which I think would be undocumented.

So I think that document the `input_urls` parameter is most important, and we can include a ref to `NetCDFtoZarrRecipe`.

Same comment on NetCDFtoZarrMultiVarSequentialRecipe.

Reply:
Fixed in latest commit.